src/components/datetimepicker/Datetimepicker.vue

```diff
@@ -21,7 +21,7 @@
         :placeholder="placeholder"
         :range="false"
         :disabled="disabled"
-        :mobile-native="mobileNative"
+        :mobile-native="mobileNative && tzOffset == 0"
```

Comment:
And this?

Reply:
Sorry, I should have explained: as far as I can tell, it is not possible to make the native API play along with this support. As such, `mobileNative` and `tzOffset` are mutually exclusive… Perhaps adding a warning would be a better way to address this.

Reply:
Yes, please add only warning

Reply:
Sure. Any model of how you'd like that warning to be handled?

Reply:
In the docs?

Reply:
Do you mean there should just be a warning in the docs? I was rather talking about a runtime warning or error, that prevents this use (otherwise the dev could end up with unpredictable behaviours without noticing).

Note that `v-if="!isMobile || inline || tzOffset != 0"` should still remain, since we don't want the code to automatically use a mobile native display if a `tzOffset` is provided…

Reply:
Ok but please add this check in s computed prop and not in the template

Reply:
OK, I added:
```
isMobileNative() {
          return this.mobileNative && tzOffset == 0
        },
```
What do you think?

Reply:
Ok